jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -101,7 +101,7 @@
           "subdir": "doc/alertmanager-mixin"
         }
       },
-      "version": "main",
+      "version": "0f1a701e9d233211cfd04033150223d404d751a3",
```

Comment:
Keeping it to commit instead of release-0.28 to get the change https://github.com/prometheus/alertmanager/commit/0f1a701e9d233211cfd04033150223d404d751a3 as well

(github doesn't show changes as it was using main which already had the change)